### DISCUSSION TITLE
Will weight files be published?

### DISCUSSION THREAD

**Discussion by KaizerJ** [fcakyon/small-object-detection-benchmark#11] [Q&A] [answered]
Will you make available the weight files for the different settings used in the paper?

**Comment by fcakyon** [comment]
@KaizerJ which weights are you interested in, I have plans to publish them but am a bit busy atm 👍

**Comment by KaizerJ** [comment]
@fcakyon I'll be fine with TOOD+SF+SAHI+FI+PO in visdrone and xview datasets. I'll like to reproduce the full setup (SF+SAHI+FI+PO) but Im still some doing preliminary experiments and cant reproduce the training by myself. Thanks

**Comment by fcakyon** [comment]
@KaizerJ I am really busy atm, will try my best to send you the files in the following days :+1:

**Comment by FabienMerceron** [comment]
Thank you for your amazing work. I would also be very interested in the weights you obtained after training on xView if possible. Thanks

**Comment by fcakyon** [comment, marked as answer]
@KaizerJ @FabienMerceron I have uploaded the TOOD+SF checkpoint files for both xView and Visdrone, check the readme for the URLs 👍 

I have also uploaded the prediction+evaluation+analysis scripts covering most of the settings in the paper. CHeck readme for details: https://github.com/fcakyon/small-object-detection-benchmark#evaluation
